src/tsconfig.app.json

```diff
@@ -3,11 +3,14 @@
   "compilerOptions": {
     "outDir": "../out-tsc/app",
     "baseUrl": "./",
-    "module": "es2015",
+    "module": "esnext",
     "types": []
   },
+  "angularCompilerOptions": {
+    "enableIvy": true
```

Comment:
Ivy support is not stable yet, this isn't a good decision

Reply:
true, but its supposed to produce a smaller bundle and it seemed to work locally (I already updated the lazyloading syntax), so it might be worth it.